lib/checks.js

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) 2012 Mathieu Turcotte
  * Licensed under the MIT license.
  */
+'use strict';
```

Comment:
Fair enough. I actually didn't think about the strict mode part, I just sort of added it out of reflex. The optimization thing was my bigger motivation. I'll update the description (and add the whitespace).